Dockerfile

```diff
@@ -22,7 +22,7 @@ COPY --from=haskell /opt/ghc/8.8.1/lib/ghc-8.8.1/* /opt/ghc/8.8.1/lib/ghc-8.8.1/
 ENTRYPOINT ["/protobuf/bin/protoc", "-I/protobuf", "--plugin=protoc-gen-haskell=/usr/local/bin/proto-lens-protoc"]
 
 # Build semantic
-FROM haskell:8.8 as build
+FROM haskell:8.8.1 as build
```

Comment:
I think I'd prefer we fix line 20 to be less of a hack as that appears to be the only thing that's hard coded a path with a ghc version in it. That part of the multistage build is so that protoc tooling works for development. Alternatively we could pin to 8.8.3 (and manually fix line 20).

@patrickt what do you think about picking a specific version of ghc in here. Is major.major enough in here or should we be strict on the minor version too?

Reply:
@tclem major.major is the only one we should be concerned about re. compatibility and whatnot; we want to choose the latest GHC 8.8. Pinning to 8.8.3 sounds fine to me.

Reply:
Fixed!